__tests__/lib/pg/service.test.js

```diff
@@ -29,11 +29,20 @@ const scpPostgresCredentials = {
   password: 'postgres',
 }
 const scpModel = './__tests__/__assets__/cap-proj/srv/'
-
+const localSchemaCredentials = {
```

Comment:
could you please integrate this into `__tests__/lib/pg/_buildSuiteEnvironments.js` ?